docs/collections/graphs/using-rest-api.md

```diff
@@ -60,193 +60,196 @@ To create `edge collection` use same endpoint `/_fabric/{fabric_name}/_api/colle
-    print("\nDocument Collection1 Created: ",result)
-
-    payload = { 'name': COLLECTION_NAME_2 }
-
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nDocument Collection2 Created: ",result)
-
-    payload = [
-        {
-            '_key':'Jean',
-            'firstname': 'Jean',
-            'lastname':'Picard',
-            'email':'jean.picard@macrometa.io'
-        },
-        {
-            '_key':'James',
-            'firstname': 'James',
-            'lastname':'Kirk',
-            'email':'james.kirk@macrometa.io'
-        },
-        {
-            '_key': 'Han',
-            'firstname': 'Han',
-            'lastname':'Solo',
-            'email':'han.solo@macrometa.io'
-        },
-        {
-            '_key': 'Bruce',
-            'firstname': 'Bruce',
-            'lastname':'Wayne',
-            'email':'bruce.wayne@macrometa.io'
-        }
-    ]
-
-    url = FED_URL + "/_api/document/" + COLLECTION_NAME_1
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nDocuments Inserted: ",result)
-
-    payload = [
-        {'_id': 'lectures/CSC101', 'difficulty': 'easy', '_key':'CSC101', 'firstname':'Jean'},
-        {'_id': 'lectures/CSC102', 'difficulty': 'hard', '_key':'CSC102','firstname':'Jean'},
-        {'_id': 'lectures/CSC103', 'difficulty': 'hard', '_key':'CSC103','firstname':'Jean'},
-        {'_id': 'lectures/CSC104', 'difficulty': 'moderate', '_key':'CSC104','firstname':'Jean'}
-
-    ]
-
-    url = FED_URL + "/_api/document/" + COLLECTION_NAME_2
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nDocuments Inserted: ",result)
-
-    # Create Edge Collection
-
-    payload = { 'name': EDGE_COLL_NAME, "type":3 }
-
-    url = FED_URL + "/_api/collection"
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nEdge Collection Created: ",result)
-    payload = [
-        {
-        '_key': 'Jean-CSC101',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC101',
-        'online': False
-        },
-        {
-        '_key': 'Jean-CSC102',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC102',
-        'online': True
-        },
-        {
-        '_key': 'Jean-CSC103',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC103',
-        'online': False
-        },
-        {
-        '_key': 'Bruce-CSC101',
-        '_from': 'teachers/Bruce',
-        '_to': 'lectures/CSC101',
-        'online': True
-        }
+""" This file is a demo on using edge collections and graphs """
+import json
+import requests
+
+# Constants
+
+FEDERATION = "api-gdn.paas.macrometa.io"
+FED_URL = f"https://{FEDERATION}"
+GEO_FABRIC = "_system"
+API_KEY = "my IP key" # Change to your API key
+#EMAIL = "my email"
+#PASSWORD = "my password"
+#AUTH_TOKEN = "bearer " # Do not change
```

Comment:
we should not keep this email,pass, authtoken commented since we are not showing the user any alternative way to use them here. if we are using APikey  then we should remove these other methods. Also API_KEY should have sample like API_KEY = "apikey <yourapikey>" since "apikey" keyword is needed and user doesn't know that. Really sorry for being so late.

Reply:
Hi. I left these available in case the user chooses that method instead of the API key way. The user should also uncomment the multi-line comment beginning at line 84-85 "url = f"{FED_URL}/_open/auth"".

The "apikey" keyword is not needed as that works on my tests.

Thanks for your comments.